thirdeye-plugins/thirdeye-notification-email/src/main/java/ai/startree/thirdeye/plugins/notification/email/EmailSendgridNotificationService.java

```diff
@@ -48,9 +48,13 @@ public EmailSendgridNotificationService(final EmailSendgridConfiguration configu
 
   @Override
   public void notify(final NotificationPayloadApi api) throws ThirdEyeException {
+    if (api.getAnomalyReports().isEmpty()) {
```

Comment:
From what I understand this will only happen if: 
- there are new completed anomalies
- there are no new anomalies 

If I understand correctly, could we make the log a bit more explicit? 
Something like: 
```
No new anomalies to notify.
```

Also, should we use a `DEBUG` level instead of info? 
The biggest workload I'm aware of is: 
~10 subscription groups
~notification task runs every 20 seconds (3 per minute)
~2 channels. (slack, webhook)
--> this could mean we generate ~60 logs like this per minute. 
Not sure this log is very helpful.
The log when there is no anomaly at all is DEBUG level: https://github.com/startreedata/thirdeye/blob/056ca48b189e2078c010439f558dfbe63341cb1f/thirdeye-worker/src/main/java/ai/startree/thirdeye/worker/task/runner/NotificationTaskRunner.java#L150

Reply:
Yes, that is the case. Makes sense to turn it into a DEBUG level log. Making this change.